examples/android-gradle/build.gradle

```diff
@@ -0,0 +1,35 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath "com.android.tools.build:gradle:$BuildPluginsVersion.AGP"
+        classpath "io.sentry:sentry-android-gradle-plugin"
+    }
+}
+
+apply plugin: 'com.android.application'
+apply plugin: 'io.sentry.android.gradle'
+
+android {
+    compileSdkVersion = 30
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled true
+        }
+    }
+}
+
+sentry {
+    autoProguardConfig = true
+    autoUpload = true
```

Comment:
here could be omitted, it's enabled by default

Reply:
let's set it to `false` right till we figure out this -> https://github.com/getsentry/sentry-android-gradle-plugin/pull/47#discussion_r584158468

Reply:
Updated 👍